src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/Employee/Blocks/form.html.twig

```diff
@@ -36,155 +36,153 @@
-        {{ form_errors(employeeForm) }}
+    <div class="card-body">
+      {{ form_errors(employeeForm) }}
 
-        {{ ps.form_group_row(employeeForm.firstname, {}, {
-          'label': 'First name'|trans({}, 'Admin.Global')
-        }) }}
+      {{ ps.form_group_row(employeeForm.firstname, {}, {
+        'label': 'First name'|trans({}, 'Admin.Global')
+      }) }}
 
-        {{ ps.form_group_row(employeeForm.lastname, {}, {
-          'label': 'Last name'|trans({}, 'Admin.Global')
-        }) }}
+      {{ ps.form_group_row(employeeForm.lastname, {}, {
+        'label': 'Last name'|trans({}, 'Admin.Global')
+      }) }}
 
-        {{ ps.form_group_row(employeeForm.avatarUrl, {}, {
-          'label': 'Avatar'|trans({}, 'Admin.Global')
-        }) }}
+      {{ ps.form_group_row(employeeForm.avatarUrl, {}, {
+        'label': 'Avatar'|trans({}, 'Admin.Global')
+      }) }}
 
-        <div class="form-group row">
-          <label class="form-control-label"></label>
-          <div class="col-sm">
-            <img class="img-thumbnail clickable-avatar" src="{{ avatarUrl }}" alt="">
-          </div>
+      <div class="form-group row">
+        <label class="form-control-label"></label>
+        <div class="col-sm">
+          <img class="img-thumbnail clickable-avatar" src="{{ avatarUrl }}" alt="">
         </div>
+      </div>
 
-        {{ ps.form_group_row(employeeForm.has_enabled_gravatar, {}, {
-          'label': 'Enable gravatar'|trans({}, 'Admin.Global')
-        }) }}
+      {{ ps.form_group_row(employeeForm.has_enabled_gravatar, {}, {
+        'label': 'Enable gravatar'|trans({}, 'Admin.Global')
+      }) }}
+
+      {{ ps.form_group_row(employeeForm.email, {}, {
+        'label': 'Email address'|trans({}, 'Admin.Global')
+      }) }}
+
+      {% set passwordHelpMessage = 'Password should be at least %num% characters long.'|trans({ '%num%': 8 }, 'Admin.Advparameters.Help') %}
+
+      {% if isRestrictedAccess %}
+          {% set oldPasswordVars = employeeForm.change_password.old_password.vars|merge(old_password|default({})) %}
+          {% set newPasswordFirstVars = employeeForm.change_password.new_password.children.first.vars|merge(new_password.first_options|default({})) %}
+          {% set newPasswordSecondVars = employeeForm.change_password.new_password.children.second.vars|merge(new_password.second_options|default({})) %}
+
+          <div class="form-group row">
+            <label class="form-control-label">
+              {{ 'Password'|trans({}, 'Admin.Global') }}
+            </label>
+            <div class="col-sm">
+              {# "Change password" button #}
+              {{ form_widget(employeeForm.change_password.change_password_button, {
+                label: 'Change password...'|trans({}, 'messages'),
+                attr: {
+                  class: 'btn-outline-secondary js-change-password',
+                }
+              }) }}
+
+              <div class="card card-body js-change-password-block d-none">
```

Comment:
I know it was already like but is it normal the `card` div and the `cart-body` are the same? I though the structure was:
```
div.card
   -> div.card-body
```

Reply:
Also, it seems weird to have a card inside another, but I guess this one is used as a template for a js component no? Or maybe a togglable card inside a bigger one?

Reply:
Perfectly fine to have nested cards